www_src/pages/page/page.jsx

```diff
@@ -157,58 +176,66 @@ var Page = React.createClass({
   deleteElement: function() {
     if(this.state.currentElement === -1) return;
     var elements = this.state.elements;
-    elements[this.state.currentElement] = false;
-    // note that we do not splice, because the updateElement
-    // function relies on immutable array indices.
-    this.setState({
-      elements: elements,
-      currentElement: -1
-    });
-  },
+    var id = elements[this.state.currentElement].id;
 
-  addLink: function() {
-    this.appendElement(Generator.generateDefinition(Generator.LINK, {
-      href: "https://webmaker.org",
-      label: "webmaker.org",
-      active: false
-    }));
-  },
+    // Don't delete test elements for real;
+    if (parseInt(id, 10) <= 3) return alert('this is a test element, not deleting.');
```

Comment:
Well... to be honest, it will stop the web view (with fake params) from being able to work. I was also going to use them to write some functional tests originally but never got around to it.